ak/ak_build.py

```diff
@@ -135,21 +136,30 @@ def _print_addons_path(self, config):
         repos without modules are added explicitely to the path"""
         spec = yaml.load(open(config).read())
         paths = [LINK_FOLDER, LOCAL_FOLDER]
+        relative_paths = []
         for repo_path, repo in spec.items():
             if not repo.get('modules'):
                 if repo_path == 'odoo':
                     # When odoo, we need to add 2 path
                     paths.append('%s/odoo/addons' % ODOO_FOLDER)
                     paths.append('%s/addons' % ODOO_FOLDER)
+                # Is it still really needed? why would we use relative path
+                # with C2C docker image? I think it should be removed
                 elif repo_path[0:2] == './':
-                    paths.append(repo_path)  # don't touch relative paths
+                    relative_paths.append(repo_path)
                 else:
                     # TODO Need to be delete when all spec.yaml files cleaned
+                    # Update 2019/04 No it should not?
                     paths.append('%s/%s' % (VENDOR_FOLDER, repo_path))
 
-        addons_path = ','.join(paths)
-        print('Addons path for your config file: ', addons_path)
-        return addons_path
+        # Construct absolute path, then relative path if any and merge both
+        # AFAIK the relative path logic should be removed.
+        # Because we do not use it and it add complexity
+        addons_path = '{}{}'.format(PREFIX, ',{}'.format(PREFIX).join(paths))
+        relative_path = ','.join(relative_paths)
+        whole_path = '{},{}'.format(addons_path, relative_path)
```

Comment:
You can achieve like that: 
```py

print_abs_path = ",".join(['/odoo/' + path for path in paths])
print('Addons path for your config file: ', print_abs_path) 
```
In that way, it's less complex and don't change the return value of this method.

Reply:
@hparfr 
What about now?
Notice, I don't print the relative path at all. As you said, if this feature is used, it is more likely for custom lib or stuff, but probably not odoo module, so it should not be in the addons_path.
Or we could make a second print if we think it is useful, personally, I don't think it is, but I don't mind printing stuff!

Also, I now return the same stuff as before, but I am not sure it is very consistent.
First the return value is never used, and ak is not so used outside akretion and even less overridden, so I don't think it is an issue to change this return value right now, but also, seems strange to print absolute path and return something else.
Actually, returning a value in a method called "print" is kind of strange also, maybe we should just remove the return value.

Reply:
>Also, I now return the same stuff as before, but I am not sure it is very consistent.
First the return value is never used, and ak is not so used outside akretion and even less overridden, so I don't think it is an issue to change this return value right now, but also, seems strange to print absolute path and return something else.
Actually, returning a value in a method called "print" is kind of strange also, maybe we should just remove the return value.
Correct.